src/components/search/SearchContainer.scss

```diff
@@ -3,6 +3,10 @@
 .search-container {
   &-input {
     text-align: center;
+
+    &-field {
+      width: 70%;
```

Comment:
I think 70% might too big - could we go with 40%?

Reply:
Sure. I've already made changes.